game_class.rb

```diff
@@ -0,0 +1,18 @@
+require './item_class'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, publish_date)
+    super(publish_date, archived)
```

Comment:
When you inherit, to initialize the child class you need to pass all the parent arguments plus the added from the child. It will look like this: 

```
def initialize(genre, author, label, publish_date, multiplayer, last_played_at) 
    super(genre, author, label, publish_date)
```

Please, check it.